Added Central, Mountain and Pacific timezones.

tonydew opened the pull request:
Looks like a huge chunk of Canada got left off...

tonydew committed:
Added Central, Mountain and Pacific timezones.

jonahpatriarche reviewed (changes requested):
I think some of these 'am's and 'pm's are off - in Pacific, for instance, jumping from Rachel@08:15am to Adam@09:15pm seems odd

tonydew commented:
I just swiped most of the code from the America list and added the additional Canada zones. I'll take a look and clear it up.

tonydew commented:
Fixed 'am' and 'pm' mistakes in usa.md and canada.md, and added missing timezones to canada.md

https://github.com/introwit/laracon-online-schedule/pull/85
https://github.com/introwit/laracon-online-schedule/pull/84
